CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Git LFS Changelog
 
+## 1.4.4 (21 October, 2016)
+
+### Bugs
+
+* commands,lfs/tq: Only send unique OIDs to the Transfer Queue #1600 (@ttaylorr)
```

Comment:
I think the package prefix is unnecessary for the changelog, as it's only relevant to lfs developers, and not end users. Up to you whether that warrants a change or not though.